preload.py

```diff
@@ -321,12 +322,21 @@ def fetch_webapp(app_url, directory=None):
 
 
 def main():
+
+    parser = argparse.ArgumentParser(description='Preload webapps.')
+    parser.add_argument('--root', default=os.getcwd(),
+                        help='Root directory to run from')
```

Comment:
and please add a line of document in `README.md` to demonstrate how this option works :notebook: